lib/ioki/apis/operator_api.rb

```diff
@@ -255,6 +255,17 @@ class OperatorApi
         base_path:   [API_BASE_PATH, 'products', :id, 'rides', :id],
         path:        'phone_calls',
         model_class: Ioki::Model::Operator::PhoneCall
+      ),
+      Endpoints.crud_endpoints(
+        :rematching_attempt,
+        base_path:   [API_BASE_PATH, 'products', :id],
```

Comment:
I added comments on the other PR - if we have top level access on those objects, we need safe scoping in triebwerk. I assume we will do that there, so i will prematurely mark this PR as "approved".